### ISSUE TITLE
combine multiple animations into a single file

### ISSUE THREAD

**Issue by fangzhangmnm** [facebookincubator/FBX2glTF#188]

**Comment by zellski** [comment]
That'd be nice. On the wishlist.

**Comment by helikopterodaktyl** [comment]
Isn't this basically https://github.com/facebookincubator/FBX2glTF/pull/179 ?

**Comment by zellski** [comment]
@helikopterodaktyl Yeah, it is. I'm pretty focused on getting 0.9.6 out now, and only looking at critical omissions and bugs, so it'll have to wait a bit. I still need to figure out how I want the multiple file thing t work. That PR is very helpful, but I don't like the magic of appending numbers to a base filename. All files should be explicitly given on the command line. That's a smallish tweak though. We'll see when 0.9.6 is out.

**Comment by zellski** [comment]
But you're right in that this can be closed; that PR is enough.

**Comment by helikopterodaktyl** [comment]
@zellski Unity engine uses a naming convention of base mesh in Foo.fbx and animations provided in Foo@Walk.fbx, Foo@Idle.fbx, Foo@Run.fbx etc. 

I don't know if it's a Unity-specific thing, but it seems like a clean solution for providing a mapping between files and animations, something FBX2GLTF could pick up on and automatically apply.

**Comment by zellski** [comment]
That's a fine naming scheme, and if people want to use that, I'm all for it, but I don't want the tool to default to slurping up stuff that just happens to have reasonable-looking names. This tool is commonly used in production pipelines, where that kind of semi-defined behaviour is frowned upon.

A precise invocation would look something like:
```
FBX2glTF --input "./Foo.bx" \
  --animation-input "./Foo@Walk.fbx" \
  --animation-input "./Foo@Run.fbx" \
  ...
```

That said, some opt-in magic would probably be fine, like maybe `--glob-animation-inputs`, in which case I agree that Unity's naming scheme looks good.
